README.md

```diff
@@ -145,13 +145,13 @@ build-using-dockerfile -t mybuildkit -f ./hack/dockerfiles/test.Dockerfile .
 docker inspect myimage
 ```
 
-##### Building a Dockerfile using [external frontend](https://hub.docker.com/r/tonistiigi/dockerfile/tags/):
+##### Building a Dockerfile using [external frontend](https://hub.docker.com/r/docker/dockerfile-upstream/tags/):
```

Comment:
dockerfile-upstream -> dockerfile?

Reply:
done. I think we need more docs around #528 to make it clear how this works and when to use each repo.